mobile-center-analytics/android/src/main/java/com/microsoft/azure/mobile/react/analytics/RNAnalyticsPackage.java

```diff
@@ -40,7 +40,7 @@ public RNAnalyticsPackage(Application application, String startEnabled) {
         return modules;
     }
 
-    @Override
```

Comment:
Would "No @Override to support application on react native 0.47.0 or later" be a better wording?